AutomatedTesting/Gem/PythonTests/Prefab/tests/create_prefab/CreatePrefab_UnderChildEntityOfAnotherPrefab.py

```diff
@@ -0,0 +1,82 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def CreatePrefab_UnderChildEntityOfAnotherPrefab():
+    """
+    Test description:
+    - Creates parent/child entities with the entities having a physx collider
+    - Creates a prefab "Outer_prefab" and an instance based of the entities
+    - Creates a prefab "Inner_prefab" inside "Outer_prefab" based the child entity contained inside of it
+
+    Checks that the parent/child entity is correctly handlded by the prefab system checking the name and that it contains the physx collider
+    """
+
+    from pathlib import Path
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.prefab_utils import Prefab
+
+    import Prefab.tests.PrefabTestUtils as prefab_test_utils
+
+    OUTER_PREFAB_NAME = 'Outer_prefab'
+    INNER_PREFAB_NAME = 'Inner_prefab'
+    OUTER_PREFAB_FILE_NAME = Path(__file__).stem + OUTER_PREFAB_NAME
+    INNER_PREFAB_FILE_NAME = Path(__file__).stem + INNER_PREFAB_NAME
+    PARENT_ENTITY_NAME = "ParentEntity"
+    CHILD_ENTITY_NAME = "ChildEntity"
+    PHYSX_COLLIDER_NAME = "PhysX Collider"
+
+    prefab_test_utils.open_base_tests_level()
+
+    # Creates new parent/child Entities at the root level
+    # Asserts if creation didn't succeed
+    parent_entity = EditorEntity.create_editor_entity_at((100.0, 100.0, 100.0), name=PARENT_ENTITY_NAME)
+    assert parent_entity.id.IsValid(), "Couldn't create parententity"
+    parent_entity.add_component(PHYSX_COLLIDER_NAME)
+    assert parent_entity.has_component(PHYSX_COLLIDER_NAME), "Attempted to add a PhysX Collider but no physx collider collider was found afterwards"
```

Comment:
I'd simplify this assert statement to something like "Failed to add a PhysX Collider"